Python/sysmodule.c

```diff
@@ -3479,6 +3479,66 @@ make_impl_info(PyObject *version_info)
     return NULL;
 }
 
+static PyObject*
+make_abi_features(void)
+{
+    int res;
+    PyObject *features = PyList_New(0);
```

Comment:
Instead of having a list here, you can do `PyFrozenSet_New(NULL)` and then use `PySet_Add(item)` to build it. It's still mutable before being exposed elsewhere.

Reply:
Ah perfect. Did not know that.